vestido/vestido-storefront/app/api/payment/[paymentSlug]/route.ts

```diff
@@ -0,0 +1,85 @@
+import { ZodError } from 'zod';
```

Comment:
Change `/api/payment/[paymentId]` to `/api/payments/[paymentId]`. Please pluralize